Source/Core/VideoCommon/Present.cpp

```diff
@@ -844,10 +844,10 @@ void Presenter::Present()
   UpdateDrawRectangle();
 
   g_gfx->BeginUtilityDrawing();
-  g_gfx->BindBackbuffer({{0.0f, 0.0f, 0.0f, 1.0f}});
+  bool ok = g_gfx->BindBackbuffer({{0.0f, 0.0f, 0.0f, 1.0f}});
```

Comment:
Minor nitpick: why is this 'ok'? I don't know what else you'd call it here though.

Reply:
Because I also didn't know what else to call it

Reply:
`backbuffer_bound`

Reply:
```suggestion
  const bool backbuffer_bound = g_gfx->BindBackbuffer({{0.0f, 0.0f, 0.0f, 1.0f}});
```